pkg/cmd/provisioning/ibmcloud/create_service_id.go

```diff
@@ -152,10 +151,10 @@ func processCredentialsRequests(client ibmcloud.Client,
 		serviceIDName := fmt.Sprintf("%s-%s-%s", CreateOpts.Name, cr.Spec.SecretRef.Namespace, cr.Spec.SecretRef.Name)
```

Comment:
For now, I think the only way of identifying the service ids created is by looking for this specific name format. Do we want to block the creation of a new service id if one with the name already exists?

Reply:
well, make sense, but what if someone creates service accounts with the same name after running this tool?! any either way we may end up deleting all those service IDs following this naming convention.

But this is going to be a little tricky because this tool runs on a directory looping through all the files and create service IDs, so will have to think about how to handle if failes: 

1. fail and keep the created service IDs?
2. fail and delete all the created service IDs?
3. run a loop through all the files and first check for the service ID existence and fail if exists

Reply:
3rd solution looks to be clean, lemme know if you have any other thought

Reply:
I agree, I am fine with the 3rd option